src/components/Dust/FineDustState.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/styled';
+
+interface FineDustStateProps {
+  fineDustState: number;
+}
+
+const FineDustState = ({ fineDustState }: FineDustStateProps) => {
+  if (fineDustState < 80)
+    return <DustStateColor color="#1E64EE">{fineDustState}좋음</DustStateColor>;
+  if (fineDustState < 150)
+    return <DustStateColor color="#00D500">{fineDustState}보통</DustStateColor>;
+  if (fineDustState < 300)
+    return <DustStateColor color="#F95A20">{fineDustState}나쁨</DustStateColor>;
+  if (isNaN(fineDustState))
+    return (
+      <DustStateColor color="#666666">{fineDustState}측정중</DustStateColor>
+    );
```

Comment:
```suggestion
   if (isNaN(fineDustState))
    return (
      <DustStateColor color="#666666">{fineDustState}측정중</DustStateColor>
    );
  if (fineDustState < 80)
    return <DustStateColor color="#1E64EE">{fineDustState}좋음</DustStateColor>;
  if (fineDustState < 150)
    return <DustStateColor color="#00D500">{fineDustState}보통</DustStateColor>;
  if (fineDustState < 300)
    return <DustStateColor color="#F95A20">{fineDustState}나쁨</DustStateColor>;
```

순서가 바뀌어야할것 같아요!  `isNaN`이 맨 위로!

근데 이부분 `<DustStateColor color={color}>{content}</DustStateColor>`이 반복되는데 리팩터링이 가능할 것같아요 ! ㅋㅋ 이건 나중에 여유되시거나 신경쓰이시면 진행해주시면 좋을 것 같습니다